sdk/provisioning/Generator/src/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Azure.Provisioning.Generator.Model;
+using Azure.Provisioning.Generator.Specifications;
+
+// Collect all the specs to generate
+List<Specification> specs =
+[
+    // Baseline
+    new ArmSpecification(),
+    new ResourcesSpecification(),
+    new AuthorizationSpecification(),
+    new ManagedServiceIdentitiesSpecification(),
+
+    new AppContainersSpecification(),
+    new AppServiceSpecification(),
+    new AppConfigurationSpecification(),
+    new ApplicationInsightsSpecification(),
+    new CommunicationSpecification(),
+    new CognitiveServicesSpecification(),
+    new ContainerRegistrySpecification(),
+    new ContainerServiceSpecification(),
+    new CosmosDBSpecification(),
+    new EventGridSpecification(),
+    new EventHubsSpecification(),
+    new KeyVaultSpecification(),
+    new KubernetesSpecification(),
+    new KubernetesConfigurationSpecification(),
+    // new NetworkSpecification(), // TODO: Resolve bicep serialization issue
+    new OperationalInsightsSpecification(),
+    new PostgreSqlSpecification(),
+    new RedisSpecification(),
+    new SearchSpecification(),
+    new ServiceBusSpecification(),
+    new SignalRSpecification(),
+    new SqlSpecification(),
+    new StorageSpecification(),
+    new WebPubSubSpecification(),
+];
+
+// Generate the specs
+Dictionary<string, string> failures = [];
+foreach (Specification spec in specs)
+{
+    try
+    {
+        Console.WriteLine($"Generating {spec.Name}...");
+        spec.Build();
+    }
```

Comment:
All of this is getting spit out of a throwaway prototype generator that reflects over our management plane APIs.  If we already had an extensible generator ready to go today, I would have absolutely implemented all this as a plugin.  There's not much code here though so it will be really easy to throw away next year when schedules are better aligned.